src/Sulu/Component/Rest/ListBuilder/Doctrine/DoctrineListBuilder.php

```diff
@@ -183,7 +184,11 @@ function ($array) {
     protected function assignSortFields($queryBuilder)
     {
         foreach ($this->sortFields as $index => $sortField) {
-            $queryBuilder->addOrderBy($sortField->getSelect(), $this->sortOrders[$index]);
+            $sort = ($sortField instanceof DoctrineConcatenationFieldDescriptor)
+                ? $sortField->getName()
+                : $sortField->getSelect();
```

Comment:
Can you maybe check the exact error? Just seeing that the status code is 500 is not really enough to argue about that :-)